### PR TITLE
set up complete time with vm clock

### PR DESCRIPTION
**Pull request by zacharym**
@jsingle cc @thieman

### CODE REVIEW

File: dusty/systems/virtualbox/__init__.py [hunk 1]
```diff
@@ -16,7 +18,12 @@
 def _run_command_on_vm(command_list):
     ssh_command = ['docker-machine', 'ssh', constants.VM_MACHINE_NAME]
     ssh_command.extend([command_list])
-    check_and_log_output_and_error_demoted(ssh_command)
+    result = check_and_log_output_and_error_demoted(ssh_command)
+    return result
+
+def vm_current_time():
```
Comment (jsingle) [inline dusty/systems/virtualbox/__init__.py]: If we're only using this for tests, this function should go in a test specific location IMO

File: dusty/systems/virtualbox/__init__.py [hunk 2]
```diff
@@ -16,7 +18,12 @@
 def _run_command_on_vm(command_list):
     ssh_command = ['docker-machine', 'ssh', constants.VM_MACHINE_NAME]
     ssh_command.extend([command_list])
-    check_and_log_output_and_error_demoted(ssh_command)
+    result = check_and_log_output_and_error_demoted(ssh_command)
+    return result
```
Comment (jsingle) [inline dusty/systems/virtualbox/__init__.py]: `return check_and_log_output_and_error_demoted(ssh_command)` is simpler

File: dusty/commands/status.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 from ..compiler.spec_assembler import get_assembled_specs
 from ..log import log_to_client
 from ..systems.docker import get_dusty_containers
-from ..systems.virtualbox import docker_vm_is_running
+from ..systems.virtualbox import docker_vm_is_running, vm_current_time
```
Comment (jsingle) [inline dusty/commands/status.py]: snip?

File: dusty/systems/virtualbox/__init__.py (hunk 1 above)
Comment (thieman) [inline dusty/systems/virtualbox/__init__.py]: +1. Additionally `dateutil` is a test-only requirement, if you put it in the app code it's going to blow up.

File: tests/integration/cli/restart_test.py [hunk 4]
```diff
@@ -21,7 +22,7 @@ def setUp(self):
         self.run_command('bundles activate bundle-a bundle-b')
         self.run_command('up')
         time.sleep(.1)
-        self.up_complete_time = datetime.datetime.utcnow()
+        self.up_complete_time = vm_current_time()
```
Comment (thieman) [inline tests/integration/cli/restart_test.py]: Should add a comment here explaining why we're doing this

File: dusty/systems/virtualbox/__init__.py [hunk 5]
```diff
@@ -16,7 +18,12 @@
...
+
+def vm_current_time():
+    result = check_output_demoted(['docker-machine', 'ssh', 'dusty', 'date "+%Y-%m-%dT%X.%s"'])
```
Comment (thieman) [inline dusty/systems/virtualbox/__init__.py]: The `dusty` part of this is stored in a constant

**Pull request merged**